module/input/Camera/data/config/Cameras/Left.yaml

```diff
@@ -6,6 +6,16 @@ settings:
   Height: 1024
   OffsetX: 0
   OffsetY: 0
-  ExposureAuto: Continuous
-  GainAuto: Continuous
-  BalanceWhiteAuto: Continuous
+  ExposureAuto: Continuous # Options: Off, Once, Continuous
+  GainAuto: Continuous # Options: Off, Once, Continuous
+  Gain: 15 # Range: 0.00 - 18.01'
+  GammaEnable: true # Set to false to adjust gamma. Default = true
+  Gamma: 0.50
+  BalanceWhiteAuto: Once # Options: Off, Once, Continuous
+  BalanceRatioSelector: Red # Options: Red or Blue. Controls the balance ratio of the selected color relative to green. Used for white balancing.
+  BalanceRatio: 2.56
+  AutoExposureEVCompensation: 0 # Range from -3(darker) to 3(lighter) with a step of 1/3. Default = 0.
+  AcquisitionFrameRateEnable: true
+  AcquisitionFrameRate: 20.00
+  SaturationEnable: false
+  Saturation: 0
```

Comment:
The majority of these seem to not be in use?

Reply:
Yeah, i left them in in case we wanted to try manual tuning.

Reply:
Maybe comment out the ones we don't use?

Reply:
There are going to be hundreds of settings you don't use, that said it might be a good idea to use arv-tool to dump them all out so you can decide what's important

Reply:
Yeah it would be good to document. I'll put that on the list.